daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/infoset/JDOMInfosetInputter.scala

```diff
@@ -17,10 +17,12 @@
 
 package org.apache.daffodil.runtime1.infoset
 
+import java.lang.{ Boolean => JBoolean }
```

Comment:
Note that if we wanted to revert all the J prefixes, ones like this should stay as we are actually using a Java boolean instead of a Scala one.